projects/vc-api-implementation.md

```diff
@@ -0,0 +1,240 @@
+
+The project is also tested by unit tests and end-to-end tests. 
+
+Integration tests are implemented based on Energy and Mobility use cases, which have historically come from Elia and BloXmove energy and mobility as use-case requirements respectively. 
+
+### Documentation 
+
+The project contains technical documentation and tutorials to facilitate understanding of the API and experiment with the implementation: https://github.com/energywebfoundation/ssi/tree/develop/apps/vc-api/docs/tutorials 
+
+Furthermore, Energy Web has documentation on VC-API and general documentation on SSI available the Energy Web gitbook: 
+
+- https://energy-web-foundation.gitbook.io/energy-web/foundational-concepts/self-sovereign-identity-introduction
+- https://energy-web-foundation.gitbook.io/energy-web/ew-dos-technology-components-2023/identity-and-access-management-iam/iam-guides/verifiable-credential-api
+
+# Alignment with the OpenWallet Foundation Mission
+_Include a statement on alignment with the [OpenWallet Foundation mission](https://openwallet-foundation.github.io/tac/governance/charter/)._
+
+The project team believes that the project is well aligned with the OpenWallet Foundation mission. 
+
+An implementation of VC API developed and hosted by the OWF would advance the adoption of interoperable and privacy enhancing digital wallets. This is because VC API is an open specification of an interface for Verifiable Credential operations, which are relevant functions for digital wallets.
+Furthermore, as an HTTP API, the project is a technology well understood by many organizations and will plausibly accelerate their efforts to develop or integrate digital wallets. 
+
+In addition, development of a VC API implementation would promote the development and proliferation of the VC API specification itself, which is an open standard related to digital wallets. 
+
+The project has been open source since its inception and aims to be inclusive, in so far as it should be usable across a wide range of domains and infrastructure requirements.
+
+# Current Code of Conduct
+_Has this project adopted a Code of Conduct? If so, please include a link to it._
+
+https://github.com/energywebfoundation/ssi/blob/develop/apps/vc-api/CODE_OF_CONDUCT.md
+
+# TAC Sponsor
+_Include the name of a sponsor from the TAC, if identified (a sponsor helps mentor projects)_
+
+The project has not yet identified a TAC sponsor.
+
+# Project License
+_Include the proposed project license (Apache 2.0 by default)._
+
+GNU General Public License 3.0 -https://github.com/energywebfoundation/ssi/blob/develop/apps/vc-api/LICENSE 
+
+# Source Control
+_If the project already exists, please provide a link to the repositories._
+
+The project source code is at the following location: https://github.com/energywebfoundation/ssi/tree/develop/apps/vc-api 
```

Comment:
It appears that [DCO](https://developercertificate.org/) has not happened on the existing repository. In order to bring this source to OWF, we will need to do one of two things:

1. bring your code by squashing all of your commits into a single first commit made against a new OWF repo with your DCO sign-off.

2. amend the commit history to include DCO sign-off for each of the commits. The Hyperledger Indy community has [documented steps to fix DCO on previous commits](https://github.com/hyperledger/indy-sdk/blob/main/docs/contributors/signing-commits.md#how-to-sign-previous-commits). Also, the [Fix DCO Guide from src-d](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) contains some different steps you can take.